surveys/models.py

```diff
@@ -50,7 +50,7 @@ class SurveyFormEntry(FormEntry):
     active = models.BooleanField(default=True)
 
     def __str__(self):
-        return self.name + " (" + str(self.type) + ")"
```

Comment:
Since Tyler's template include the `type` in the `name`, and since Fobi restricts templates with the same name created by the same user, I've dropped the `type` here

Reply:
This seems reasonable to me! We'll just have to remember to include the survey type in the name of any new templates.

Reply:
Yep!